ocaml/xapi/xapi_pool.ml

```diff
@@ -3952,3 +3952,49 @@ let put_bundle_handler (req : Request.t) s _ =
       | None ->
           ()
   )
+
+let start_ssh ~__context ~self:_ =
```

Comment:
These two functions are similar. It should be possible to abstract a common one from them.